pilot/pkg/model/config.go

```diff
@@ -533,7 +533,7 @@ var (
 		ClusterScoped: false,
 		Type:          "service-role-binding",
 		Plural:        "service-role-bindings",
-		Group:         "config",
+		Group:         "rbac",
```

Comment:
Could you also change the group name of RbacConfig in line 547? It's used to configure the global RBAC policy.

Reply:
+1